searx/webapp.py

```diff
@@ -761,6 +761,11 @@ def search():
         )
     )
 
+    # timings: get engine response times sorted from slowest to fastest
+    timings = webutils.get_engine_timings(result_container.get_timings())
+    # first entry in list is the slowest engine. [0] on object is engine name, [1] is total time
+    max_response_time = timings[0][1] if len(timings) > 0 else None
```

Comment:
To avoid this additional explaination here, it might be easier to just move the sorting code here and map it into pairs afterwards, i.e.
```suggestion
    # engine_timings: get engine response times sorted from slowest to fastest
    engine_timings =  sorted(result_container.get_timings(), reverse=True, key=lambda e: e.total)
    max_response_time = timings[0].total if len(timings) > 0 else None
    engine_timings_pairs = [(timing.engine, timing.total) for timing in timings]
```